tests/test_hardware.py

```diff
@@ -37,21 +39,28 @@ async def test_update_register(self):
         self.cpu8000.fault_register.commander = unittest.mock.AsyncMock()
         await self.cpu8000.update_register()
 
-    def test_set_simulation_mode(self):
-        self.cpu8000.set_simulation_mode(True)
-        self.assertEqual(self.cpu8000.power_register.simulation_mode, True)
-        self.assertEqual(self.cpu8000.display_current_register.simulation_mode, True)
-        self.assertEqual(self.cpu8000.fault_register.simulation_mode, True)
-        with self.subTest("Test going out of simulation mode"):
-            self.sim_cpu8000.set_simulation_mode(False)
-            self.assertEqual(self.sim_cpu8000.power_register.simulation_mode, False)
-            self.assertEqual(
-                self.sim_cpu8000.display_current_register.simulation_mode, False
-            )
-            self.assertEqual(self.sim_cpu8000.fault_register.simulation_mode, False)
-
     def test_repr(self):
-        self.assertEqual(
-            repr(self.cpu8000),
-            "CPU8000:\n Power: None\n Display Current: None\n Fault code: None\n",
+        assert (
+            repr(self.cpu8000)
+            == "CPU8000:\n Power: None\n Display Current: None\n Fault code: None\n"
         )
+
+
+class TestMaxiOPG(unittest.IsolatedAsyncioTestCase):
+    def test_scu_configuration(self):
+        self.maxiopg = MaxiOPG(None, configuration="SCU")
```

Comment:
Please use the enum here.

Consider also using the enums below in the accepted_values tests, unless this make the test meaningless, e.g.:
```
accepted_values = [item.value for item in NoSCU]
```